test/framework/kogitoapp.go

```diff
@@ -187,14 +131,13 @@ func DeployServiceFromExampleFile(namespace, exampleFile string) error {
 	}
 
 	// Create basic KogitoApp stub
-	kogitoApp := getKogitoAppStub(namespace, "name-should-be overwritten-from-yaml", nil)
+	kogitoApp := GetKogitoAppStub(namespace, "name-should-be overwritten-from-yaml")
 
 	// Apply content from yaml file
 	if err := yaml.NewYAMLOrJSONDecoder(strings.NewReader(yamlContent), len([]byte(yamlContent))).Decode(kogitoApp); err != nil {
 		return fmt.Errorf("Error while unmarshalling file: %v ", err)
 	}
 
-	// Apply custom image configuration
 	setupBuildImageStreams(kogitoApp)
```

Comment:
shouldn't we put that in `GetKogitoAppStub` in fact ?

Reply:
The point is that some yaml examples can declare image versions.

What should we do in such case? Test with images declared in yaml or test with images declared in test config?

Reply:
Ok I see your point. yeah should be let like that then :)